transformer_engine/common/recipe/__init__.py

```diff
@@ -351,3 +358,41 @@ def __repr__(self) -> str:
             f"fp8_dpa={self.fp8_dpa}, "
             f"fp8_mha={self.fp8_mha}"
         )
+
+
+@dataclass()
+class HybridNVFP4BlockScaling(Recipe):
```

Comment:
What do you think of removing `Hybrid` from recipe name (`HybridNVFP4BlockScaling`) or/and making it more specific?
It may be the case that we will have more hybrid recipes in the future, possibly even NVFP4 plus something else (not MXFP8).